### PR TITLE
v5.9.6

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@jakirkham here

**Comment by jakirkham** [comment]
Thanks Matt! 🙏

Could you please cherry-pick this commit ( https://github.com/conda-forge/psutil-feedstock/commit/3e57a0c521e4d798b5428b5f6942df6bdd434d08 )? That should fix the macOS build failures

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@jakirkham now only failing on this pypy-win build

**Comment by jakirkham** [comment]
Thanks Matt! 🙏

Glad to see that worked 😄

Looking upstream issue ( https://github.com/giampaolo/psutil/issues/2325 ) appears relevant. If so, [this upstream patch]( https://github.com/giampaolo/psutil/commit/ccdb775abe50125a845362fd0b65f1189c02e78b.patch ) might be useful to add (note that is just upstream commit https://github.com/giampaolo/psutil/commit/ccdb775abe50125a845362fd0b65f1189c02e78b rendered as a patch)

**Comment by thewchan** [comment]
@jakirkham I might need a little more handholding here I apologize; I'm not sure what exactly to do here? Do I copy and paste the code you noted as a patch? And apply just to pypy 3.9 for windows?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,9 +8,11 @@ package:
 source:
   url: https://pypi.io/packages/source/p/psutil/psutil-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - 0001-Add-missing-includes-for-macOS.patch
```
Comment (jakirkham) [inline recipe/meta.yaml:12]: Yeah would just download that patch into the `recipe` directory, add it to this list, and commit those changes together

Comment (thewchan) [reply recipe/meta.yaml:12]: okay I'll give it a shot

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,9 +8,12 @@ package:
...
+    - 0002-upstream-patch-for-pypy.patch  # [python_impl == 'pypy']
```
Comment (jakirkham) [inline recipe/meta.yaml:13]: It's ok to apply this generally as upstream did as well

```suggestion
    - 0002-upstream-patch-for-pypy.patch
```

**Review by jakirkham — COMMENTED** [review]
Thanks Matt! 🙏

Added a small change below. Will add automerge so it merges after CI passes

**Comment by thewchan** [comment]
@jakirkham I added the selector because it seems every cpython build is failing but they were passing previously

**Comment by thewchan** [comment]
@jakirkham wait... never mind something else is going on here

**Comment by jakirkham** [comment]
Going to try restarting. The error looked a little weird. Something about being unable to get the source?

File: recipe/0002-upstream-patch-for-pypy.patch [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+   clearer tracebacks when exceptions are raised by psutil.
+ 
++**Bug fixes**
++
++- 2325_, [PyPy]: psutil did not compile on PyPy due to missing
++  `PyErr_SetExcFromWindowsErrWithFilenameObject` cPython API.
++
+ 5.9.6
+ =====
+
```
Comment (jakirkham) [inline recipe/0002-upstream-patch-for-pypy.patch:27]: Let's drop this part of the patch. Looks like it causes conflicts. Besides it is just updating the changelog, which don't need to worry about here

**Review by jakirkham — COMMENTED** [review]
Ok think I understand what is happening

Am seeing the following [in the CI log]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=873090&view=logs&jobId=a3e8a21a-cf17-5483-5dfe-b752375c22e3&j=aaada960-d85f-5b88-82e5-99df4d27a2ce&t=0aca0cbb-1f96-540a-bae4-6a7030c11d7d ):

```
checking file HISTORY.rst
Hunk #1 FAILED at 10.
1 out of 1 hunk FAILED
```

Think this means the patch is conflicting when applied. Would suggest we just drop that part of the patch as we don't need it. Have indicated the lines below

That should fix the issue

**Comment by jakirkham** [comment]
Thanks Matt! 🙏

**Comment by thewchan** [comment]
Thank YOU @jakirkham for all you help!

**Comment by jakirkham** [comment]
Happy to help 🙂

**Pull request merged**